### ISSUE TITLE
Show closeup of carbuncles

### ISSUE THREAD

**Issue by ketch** [ketch/circular_hydraulic_jump#12]
I think we should show a zoomed plot of a single carbuncle in Figure 6 and Figure 7(a), to make it clear that the instabilities there are carbuncles while the ones in Figure 7(b) and Figure 8 are not.  I think that plotting the x- or y-component of the velocity (depending on the location of the carbuncle) can be effective in showing this, since there is usually a 1-cell-wide region with anomalous velocity at the tip of the carbuncle.

**Comment by manuel-quezada** [comment]
I am trying to handle this issue with the new PR. Once that PR is merged we can close this issue.
